packages/lumos/src/index.ts

```diff
@@ -46,3 +46,4 @@ export { CellCollector, Indexer } from "@ckb-lumos/ckb-indexer";
 export * as helpers from "@ckb-lumos/helpers";
 export * as commons from "@ckb-lumos/common-scripts";
 export { BI } from "@ckb-lumos/bi";
+export * as molecule from "@ckb-lumos/molecule";
```

Comment:
```suggestion

```

the `molecule` may be confused with `codec.molecule`, we should find another better name